### PR TITLE
[4.0] Extension update - mobile

### PR DESCRIPTION
**Pull request by brianteeman**
Fix the mobile view for the extension updates page by replacing a non-existant class `hidden-sm` and tweaked which columns to display

For testing you can use the fake extension provided by @nikosdion here http://updates.myoldsite.com/file_null-1.0.zip

### Before
image 
image

### After
image

### CODE REVIEW

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on d8d284f41ce4e5051a9bdb01f2945e1ecc5953b7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26769">issues.joomla.org/tracker/joomla-cms/26769</a>.</sub>

**Comment by brianteeman** [comment]
@wilsonge the drone errors might be valid errors with our build script. I dont have enough access to confirm

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on d8d284f41ce4e5051a9bdb01f2945e1ecc5953b7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26769">issues.joomla.org/tracker/joomla-cms/26769</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26769">issues.joomla.org/tracker/joomla-cms/26769</a>.</sub>

File: administrator/components/com_installer/tmpl/update/default.php [hunk 1]
```diff
@@ -88,19 +88,19 @@
 											<?php echo $item->description; ?>
 										</div>
 									</th>
-									<td class="center">
+									<td class="center d-none d-md-table-cell">
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/update/default.php:91]: `center` can be removed. Same in the next cell.

**Comment by dgrammatiko** [comment]
@brianteeman @wilsonge not the project build tools but the version of node-sass:
```

Building: /usr/local/bin/node /********/src/node_modules/node-gyp/bin/node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=
--
24 | gyp info it worked if it ends with ok

.
.
.

42 | gyp verb `which` failed Error: not found: python2
```

In plain text python is missing

**Comment by infograf768** [comment]
Tks.

**Comment by brianteeman** [comment]
thanks

**Pull request merged**